auth_quick_master/controllers/main.py

```diff
@@ -15,11 +15,11 @@ class AuthQuickMaster(http.Controller):
     @http.route('/auth_quick_master/get-token', type="http", auth='user')
     def get_token(self, build, build_user_id, build_login, build_url):
         _logger.debug('Request for token: build reference = %s, build_user_id = %s, build_login = %s, build_url = %s', build, build_user_id, build_login, build_url)
-        token_obj = request.env['auth_quick_master.token'].create({
+        token_obj = request.env['auth_quick_master.token'].sudo().create({
```

Comment:
:-1: It allows anyone get access to any build. Revert this and check https://github.com/it-projects-llc/saas-addons/blob/12.0/auth_quick_master/doc/index.rst#configuration

Reply:
I get an access error when run it previous way

Reply:
You need to figure out why. You cannot add ``sudo()`` everywhere

Reply:
`sudo()` was here before the commit, but it was after `create()`, so I decided that this method is used incorrectly

Reply:
>  It allows anyone get access to any build. Revert this and check https://github.com/it-projects-llc/saas-addons/blob/12.0/auth_quick_master/doc/index.rst#configuration

I don't think so. Auth quick modules use `sudo()` everywhere in controllers but you can't login to build database without logging in to the master database

Reply:
Ok, tell me how to allow some users to  use this feature and don't allow to others

Reply:
I fixed this piece of code